### DISCUSSION TITLE
ScriptReference/SceneManagement.SceneManager.GetSceneByName

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#661] [Notes]
# ScriptReference/SceneManagement.SceneManager.GetSceneByName

https://docs.unity3d.com/ScriptReference/SceneManagement.SceneManager.GetSceneByName.html

**Comment by andreasIsCoding** [comment]
To get a [SceneAsset](https://docs.unity3d.com/ScriptReference/SceneAsset.html) by it's path use `AssetDatabase.LoadAssetAtPath <SceneAsset> (scene_path)`.  Note that you need the FULL path, and not just the scene name (Editor only, tested in 2020.3.48f).
